qutebrowser/config/configdata.yml

```
@@ -1780,6 +1784,11 @@ tabs.wrap:
   type: Bool
   desc: Wrap when changing tabs.
 
+tabs.tree_tabs:
+  default: false
+  type: Bool
+  desc: Enable tree-tabs
```

```suggestion
  desc: Enable tree-tabs.
```

:wink:

Also, maybe so more explanation (*e.g.* referring to `tabs.title.format`) might be useful here.